ios/Codeforces Watcher/Application/features/problems/cells/ProblemFiltersView.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+import common
+
+struct ProblemFiltersView: View {
+    
+    var filterItems: [ProblemFiltersViewController.UIModel] = []
```

Comment:
`UIModel` should be rather embedded in `ProblemFiltersView` (UI itself) rather than `UIViewController`.